mon-pix/app/models/assessment.js

```diff
@@ -30,15 +30,15 @@ export default class Assessment extends Model {
   @equal('type', 'COMPETENCE_EVALUATION') isCompetenceEvaluation;
   @equal('type', 'DEMO') isDemo;
   @equal('type', 'PREVIEW') isPreview;
-  @equal('type', 'SMART_PLACEMENT') isSmartPlacement;
+  @equal('type', 'CAMPAIGN') isCampaignEvaluation;
```

Comment:
```suggestion
  @equal('type', 'CAMPAIGN') isCampaign;
```
ou
```suggestion
  @equal('type', 'CAMPAIGN') isTypeCampaign;
```

Reply:
@lisequesnel J'ai eu un doute sur ce renommage : le `isCampaign` me parait gênant car ce n'est pas une campagne, c'est une évaluation pour une campagne.
Le `isTypeCampaign` me parait plus intéressant, mais je trouve cela dommage de mettre 'type' dans la dénomination sachant que c'est + un concept de la base de données qu'un concept métier

Reply:
ou alors le `isForCampaign`, le même que côté back. Je trouve ça pas déconnant d'être cohérent sur notre `is...` côté front et back, le getter fait la même chose.

Reply:
Ok pour is `isForCampaign` !